tests/verify-pool-members-vdo.yml

```diff
@@ -0,0 +1,6 @@
+- name: Validate pool member VDO settings
+  include_tasks: verify-pool-member-vdo.yml
+  loop: "{{ storage_test_pool.volumes }}"
```

Comment:
Interesting. We have to look at pool members for VDO info? So, then, is it even possible to have some vdo and some non-vdo volumes in the same pool/vg?

Reply:
After closer inspection I found that the filename here is just misleading. It is querying the pool device -- not the members.

Reply:
The README says only one vdo volume is allowed per pool, so do we really need this loop?

Reply:
There can be multiple volumes in the pool with max one of them being VDO. I am looping through volumes to get to the one with VDO on it.